### PR TITLE
UI and UX improvements for increased accessibility

### PR DESCRIPTION
**Pull request by LorenzoAncora**
## Options page

- **Improve the accessibility**: ensures that the font is readable and the interactive elements are identified as such in the options page. The buttons will be highlighted on mouse hover and on keyboard focus, while in the same circumstances the links will simply lose their underline to signify that interaction is possible;
- **Correct the bindings for `input`/`label`**: improves accessibility by ensuring correct logical relationships and easier access. Users will be able to use the linked labels to interact with the checkboxes;
- **Show the number of blocked websites**: allows for better handling of long lists and allows text-to-speech tools to correctly describe the list when it is empty.

## Options popup

- **Improve the accessibility**: the buttons and the cog icon will be highlighted on mouse hover and on keyboard focus.

### CODE REVIEW

**Comment by LorenzoAncora** [comment]
@raicem is this repository still maintained?

File: src/options/options.css [hunk 1]
```diff
@@ -97,6 +101,10 @@ h5 {
   background-color: #f7f7f9;
 }
 
+.button:hover, .button:focus {
+   filter: invert(1.0);
```
Comment (raicem) [inline src/options/options.css:105]: Can this be changed to something else in the future? I think it works for the purposes but I found it looks a bit weird with the dark blue background and the black text.

https://user-images.githubusercontent.com/10389957/150677390-99c4ba38-edf9-46fb-bccb-10925c04d12e.mp4

**Comment by raicem** [comment]
Thank you @LorenzoAncora! I'll merge this to `master` but I'll then apply some fixes (our linters doesn't really like [this bit of code](https://github.com/raicem/impulse-blocker/pull/74/files#diff-ca724fa216bf4681f64ff64dfa851e6c7cb100bdf6b77ba2cb374a8fa8a45ac9R109-R112))

File: src/options/options.css (hunk 1 above)
Comment (LorenzoAncora) [reply src/options/options.css:105]: The final appearance depends on the screen and driver settings of the end users. Regardless, the code is designed to ensure proper feedback in any situation.

**Comment by LorenzoAncora** [comment]
> Thank you @LorenzoAncora! I'll merge this to `master` but I'll then apply some fixes (our linters doesn't really like [this bit of code](https://github.com/raicem/impulse-blocker/pull/74/files#diff-ca724fa216bf4681f64ff64dfa851e6c7cb100bdf6b77ba2cb374a8fa8a45ac9R109-R112))

You're welcome. A lot of time has passed and linters were updated. If you think your linter is right, feel free to follow its advice. If the issue is a bug, please also comment in GitHub code review so I can correct myself.

**Pull request merged**